src/server/scripts/EasternKingdoms/ZulGurub/boss_mandokir.cpp

```diff
@@ -339,6 +479,76 @@ class npc_ohgan : public CreatureScript
     }
 };
 
+struct npc_chained_spirit : public ScriptedAI
+{
+public:
+    npc_chained_spirit(Creature* creature) : ScriptedAI(creature)
+    {
+        instance = me->GetInstanceScript();
+        me->AddUnitMovementFlag(MOVEMENTFLAG_HOVER);
+        me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
```

Comment:
this should be added to the DB if it's always suppose to have it 👍

Reply:
will do

Reply:
(I unresolved because I want to teach a little)

We have helpers, so instead of `SetFlag(UNIT_FIELD_FLAGS` we now have `SetUnitFlags(` :D